src/components/selectRangeUsePlanPage/SortableAgreementTable.js

```diff
@@ -151,6 +158,98 @@ const useStyles = makeStyles(theme => ({
   }
 }))
 
+function PlanRow({ agreement, location, user, currentPage }) {
+  const classes = useStyles()
+  const [open, setOpen] = useState(false)
+  return (
+    <>
+      <TableRow className={classes.root} hover tabIndex={-1} key={agreement.id}>
+        <TableCell align="left" style={{ minWidth: 150 }}>
+          {agreement.forestFileId}
+          {agreement?.plans[0]?.id && (
+            <IconButton
+              aria-label="expand row"
+              size="small"
+              style={{ marginLeft: '3px' }}
+              onClick={() => setOpen(!open)}>
+              {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+            </IconButton>
+          )}
+        </TableCell>
+
+        <TableCell align="left">
+          {agreement.plans[0]?.rangeName ?? '-'}
+        </TableCell>
+        <TableCell align="left">
+          {
+            agreement.clients.find(client => client.clientTypeCode === 'A')
+              ?.name
+          }
+        </TableCell>
+
+        <TableCell align="left">
+          {agreement.zone?.user
+            ? `${agreement?.zone?.user?.givenName} ${agreement?.zone?.user?.familyName}`
+            : 'Not provided'}
+        </TableCell>
+        <TableCell align="left">
+          {agreement.plans[0] ? (
+            agreement.plans[0]?.status.code
+          ) : (
+            <span>-</span>
+          )}
+        </TableCell>
+        <TableCell align="left">
+          {agreement.plans[0] ? (
+            <Status user={user} status={agreement.plans[0]?.status} />
+          ) : (
+            <span>-</span>
+          )}
+        </TableCell>
+        <TableCell>
+          {agreement.plans.length === 0 ? (
+            canUserEdit(PLAN.ADD, user) ? (
+              <NewPlanButton agreement={agreement} />
+            ) : (
+              <div style={{ padding: '6px 16px' }}>No plan</div>
+            )
+          ) : (
+            <Button
+              fullWidth
+              variant="outlined"
+              component={Link}
+              to={{
+                pathname: `${RANGE_USE_PLAN}/${agreement.plans[0].id}`,
+                state: {
+                  page: currentPage,
+                  prevSearch: location.search
+                }
+              }}
+              endIcon={
+                canUserEditThisPlan(agreement.plans[0], user) ? (
+                  <EditIcon />
+                ) : (
+                  <ViewIcon />
+                )
+              }>
+              {canUserEditThisPlan(agreement.plans[0], user)
+                ? 'Edit'
+                : strings.VIEW}
+            </Button>
+          )}
+        </TableCell>
+      </TableRow>
+      {agreement?.plans.length > 0 && (
+        <VersionsDropDown
+          open={open}
+          match={{
+            params: { planId: agreement?.plans[0]?.id }
+          }}></VersionsDropDown>
```

Comment:
Is there a reason we're passing `planId` via `match.params`? I think we can just use a `planId` prop directly.

Reply:
Agreed, implemented here: 7337da7